@kiva/kv-components/vue/stories/KvAccordionItem.stories.js

```diff
@@ -0,0 +1,24 @@
+import KvAccordionItem from '../KvAccordionItem.vue';
+
+export default {
+	title: 'KvAccordionItem',
+	component: KvAccordionItem,
+};
+
+const DefaultTemplate = () => ({
+	components: { KvAccordionItem },
+	template: `
+		<div style="padding: 20px;">
+			<kv-accordion-item id="accordian-test">
+				<template #header>
+					<h2>Accordion</h2>
+				</template>
+				<p>
+					"Hello, KvAccordion Contents!"
+				</p>
+			</kv-accordion-item>
+		</div>
+	`,
+});
+
+export const Default = DefaultTemplate.bind({});
```

Comment:
@tiarascoleman I just realized there are some additional stories with variations for the KvAccordianItem in the UI repo. Could you please add these additional stories here as well?

https://github.com/kiva/ui/blob/main/.storybook/stories/KvAccordionItem.stories.js

Reply:
This could become part of your "testing" ticket. I've updated the spec in VUE-1247